features/authentication.feature

```diff
@@ -0,0 +1,19 @@
+Feature: User Authentication
+  Scenario: User successfully authenticates
+    Given I have an account on the site
+    And My account is confirmed
+    When I visit the site root path
+    Then I am presented with a login page
+    When I enter my correct email
+    And I enter my password
+    And I click the "Log in" button
+    Then I am redirected to the book index page
+
+  Scenario: User enters invalid authentication information
+    Given I have an account on the site
+    And My account is confirmed
+    When I visit the site root path
+    Then I am presented with a login page
+    When I enter the wrong password
+    And I click the "Log in" button
+    Then I am notified that my password is incorrect
```

Comment:
your feature files read really nicely!

derek put together a nice guide on cucumber that includes some styling suggestions:
https://github.com/smashingboxes/guides/blob/feature/cucumber-guide/qa/cucumber.md